web/base_viewer.js

```diff
@@ -120,7 +120,7 @@ function isValidAnnotationEditorMode(mode) {
  *   being rendered. The constants from {@link AnnotationMode} should be used;
  *   see also {@link RenderParameters} and {@link GetOperatorListParameters}.
  *   The default value is `AnnotationMode.ENABLE_FORMS`.
- * @property {boolean} [annotationEditorMode] - Enables the creation and editing
+ * @property {number} [annotationEditorMode] - Enables the creation and editing
```

Comment:
While unrelated to the current patch, it fixes something I forgot to update in PR #15113.